app/trip/components/trip-form.tsx

```diff
@@ -147,8 +151,43 @@ export const TripForm = ({ tripDetails, tags, tripTags }: TripFormProps) => {
         <FormControl isInvalid={!!errors.image_url}>
           <FormLabel>Image</FormLabel>
           <HStack gap={{ base: '20px', md: '34px' }}>
-            <Image alt="" src={tripDetails?.image || imageSrc} width="50%" />
-            <PrimaryButton variant="outline">Select Image </PrimaryButton>
+            {selectedImage ? (
+              <Image
+                alt="Selected Image"
+                src={URL.createObjectURL(selectedImage)}
+                width="50%"
+                objectFit="cover"
+              />
+            ) : (
+              <Image
+                alt="Default Image"
+                src={tripDetails?.image || imageSrc}
+                width="50%"
+                objectFit="cover"
+              />
+            )}
+            <Controller
+              name="uploaded_image_file"
+              control={control}
+              render={({ field: { onChange } }) => (
+                <PrimaryButton variant="outline" as="label">
```

Comment:
Can you add cursor:pointer for hover?

Reply:
Thx! I did.
https://github.com/tabi-memo/frontend/pull/32/commits/3d3237cae33ee00f1163411a9cd512d4da6e3490#diff-16197c3b76acbf9dfbb1182d34e1a7aa8f25beff37b8bbd1080e041960183ff2R173